spec/requests/cache_spec.rb

```diff
@@ -182,6 +179,15 @@ module DisabledAnonymousAPI
   end
 
   context 'when anonymously accessed' do
+    describe '/users/alice' do
+      it 'redirects with proper cache header', :aggregate_failures do
+        get '/users/alice'
+
+        expect(response).to redirect_to('/@alice')
+        expect(response.headers['Vary']&.split(',')&.map { |x| x.strip.downcase }).to include('accept')
```

Comment:
This snippet - `response.headers['Vary']&.split(',')&.map { |x| x.strip.downcase }` appears a bunch of times in file, would make nice private method.